terraform/gitops/generate-files/templates/pm4ml/values-admin-portal.yaml.tpl

```diff
@@ -0,0 +1,90 @@
+global:
+  keto:
+    readURL: ${keto_read_url}
+    writeURL: ${keto_write_url}
+  rolePermOperator:
+    apiSvc:
+      host: ${bof_role_perm_operator_host}
+      port: 80
+  keycloak:
+    url: 'https://${keycloak_fqdn}'
+    user: '${role_assign_service_user}'
+    secret:
+      name: '${role_assign_service_secret}'
+      key: '${role_assign_service_secret_key}'
+    realm: '${keycloak_realm_name}'
+
+## TODO: Disabling the tests by default for now. Need to figure out how to configure the tests.
+## RBAC Tests
+rbacTests:
+  enabled: false
+
+## Report Tests
+reportTests:
+  enabled: false
+
+## Backend API services
+
+role-assignment-service:
+  enabled: true
+  ingress:
+    enabled: false
+  configFiles:
+    default.json: {
+        "ROLES_LIST": [
+          "pm4mladmin"
+        ],
+        "AUTO_GRANT_PORTAL_ADMIN_ROLES": [
+          "pm4mladmin"
+        ]
+      }
+
+reporting-hub-bop-api-svc:
+  enabled: false
+
+reporting-legacy-api:
+  enabled: false
+
+reporting-events-processor-svc:
+  enabled: false
+
+reporting-hub-bop-experience-api-svc:
+  enabled: false
+
+## Front-end UI services
+reporting-hub-bop-shell:
+  enabled: true
+  ingress:
+    enabled: false
+
+  config:
+    env:
+      AUTH_MOCK_API: false
+      REMOTE_API_BASE_URL: ''
+      REMOTE_MOCK_API: false
+      LOGIN_URL: https://${auth_fqdn}/kratos/self-service/login/browser?return_to=https://${portal_fqdn}
```

Comment:
here you can remove the return_to, the shell does this automatically here https://github.com/mojaloop/reporting-hub-bop-shell/blob/9b8f5f8c3f95c2f08b081a930798917f517a94f1/src/Auth/sagas.ts#L25

Reply:
removed